engine/testabilities/fixture_engine.go

```diff
@@ -117,8 +113,13 @@ func (f *engineFixture) ConfigForTests(opts ...ConfigOpts) *config.AppConfig {
 	return configuration
 }
 
-// initDbConnection creates a new connection that will be used as connection for engine
-func (f *engineFixture) initDbConnection() {
+// prepareDBConfigForTests creates a new connection that will be used as connection for engine
+func (f *engineFixture) prepareDBConfigForTests() {
+	if f.config.Db.Datastore.Engine != datastore.SQLite {
+		panic("Other datastore engines are not supported in tests (yet)")
+	}
```

Comment:
It is check if the passed configuration doesn't want to use Postgres,
because we don't support postgres in tests yet (the `env` solution should be used only for debugging and development)